azure/services/managedclusters/managedclusters.go

```diff
@@ -18,37 +18,25 @@ package managedclusters
 
 import (
 	"context"
-	"fmt"
-	"net"
-	"time"
 
 	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2021-05-01/containerservice"
-	"github.com/Azure/go-autorest/autorest/to"
-	"github.com/google/go-cmp/cmp"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/klog/v2"
-	infrav1alpha4 "sigs.k8s.io/cluster-api-provider-azure/api/v1beta1"
+	infrav1 "sigs.k8s.io/cluster-api-provider-azure/api/v1beta1"
 	"sigs.k8s.io/cluster-api-provider-azure/azure"
-	"sigs.k8s.io/cluster-api-provider-azure/azure/converters"
-	"sigs.k8s.io/cluster-api-provider-azure/util/maps"
+	"sigs.k8s.io/cluster-api-provider-azure/azure/services/async"
+	"sigs.k8s.io/cluster-api-provider-azure/util/reconciler"
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
-const serviceName = "managedclusters"
-
-var (
-	defaultUser     = "azureuser"
-	managedIdentity = "msi"
-)
+const serviceName = "managedcluster"
```

Comment:
Is there a reason we're renaming this service? If so, do we want to rename the folder under `azure/services` for consistency?

Reply:
If I remember correctly, this is just so logs are a bit more coherent since we're only creating one managed cluster per AzureManagedControlPlane, so instead of printing "managedclusters creating or updating" it will print "managedcluster creating or updating" in Conditions. Not a huge deal, I didn't want to rename the package since that's a bigger change but I can revert it if you think it's not worth it. I think groups are the same if that helps.